front-end/src/components/UploadForm.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+function UploadForm(){
+  const [verificationToken, setVerificationToken] = useState("");
+
+  function validToken() {
+    const validTokenCombination = /^[A-Z]+$/;
+    const validTokenLength = 4;
+    if(verificationToken.match(validTokenCombination) && verificationToken.length === validTokenLength)
```

Comment:
@cskeogh I have addressed the maintainability issue by creating a new function that checks the validity of the token and call it whenever I need to check the validity of the token in the HTML element